frontend/src/components/account/SideMenu/SideMenu.jsx

```diff
@@ -7,7 +7,8 @@ import { logout } from '../../../redux/slices/userSlice';
 const SideMenu = () => {
   const dispatch = useDispatch();
 
-  const handleLogout = () => {
+  const userLogout = () => {
+    localStorage.clear();
```

Comment:
Are you sure it needs to be cleared? There are not just login information

Reply:
corrected to clear just user info